lib/ansible/modules/storage/netapp/na_elementsw_access_group.py

```diff
@@ -216,8 +216,8 @@ def get_account_id(self):
         # Validate account id
         # Return account_id if found, None otherwise
         try:
-            account_id = self.elementsw_helper.account_exists(self.account_id)
-            return account_id
+            self.account_id = self.elementsw_helper.account_exists(self.account_id)
```

Comment:
The issue is really in line 233

volume_id = self.elementsw_helper.volume_exists(volume, self.account_id)

it should be

volume_id = self.elementsw_helper.volume_exists(volume, self.account_id_valid)

as we are trying to preserver self.account_id for an error message.   We want to preserve the original value.

(I agree it's a bit confusing as written, and could be better to use a new name for the original value)